recipes/popdel/Makefile.patch

```diff
@@ -16,9 +16,9 @@
  LDLIBS=-lz -lpthread
 
 -DATE=on $(shell git log --pretty=format:"%cd" --date=iso | cut -f 1,2 -d " " | head -n 1)
--VERSION=1.0.4-$(shell git log --pretty=format:"%h" --date=iso | head -n 1)
-+DATE=on 2018-11-09
-+VERSION=1.0.4
+-VERSION=1.0.5-$(shell git log --pretty=format:"%h" --date=iso | head -n 1)
++DATE=on 2018-12-06
++VERSION=1.0.5
```

Comment:
Changing the `Makefile.patch` every time the version is bumped is awkward.

Try using this from `build.sh`:

```
make VERSION=${VERSION} DATE=$(date +%Y-%m-%d)
```

Reply:
Thank you for the advice. Changing the Makefile.patch every time was indeed a bit annoying.